lnrpc/walletrpc/walletkit_server.go

```diff
@@ -304,7 +318,7 @@ func (w *WalletKit) SendOutputs(ctx context.Context,
 	// Now that we have the outputs mapped, we can request that the wallet
 	// attempt to create this transaction.
 	tx, err := w.cfg.Wallet.SendOutputs(
-		outputsToCreate, chainfee.SatPerKWeight(req.SatPerKw),
+		outputsToCreate, chainfee.SatPerKWeight(req.SatPerKw), req.Label,
```

Comment:
Looks like this RPC (`walletkit.SendOutputs`) should also check for a valid label?